opcua/server/history_sql.py

```diff
@@ -110,7 +111,7 @@ def read_node_history(self, node_id, start, end, nb_values):
 
             # select values from the database; recreate UA Variant from binary
             try:
-                for row in _c_read.execute('SELECT * FROM "{tn}" WHERE "SourceTimestamp" BETWEEN ? AND ? '
+                for row in _c_read.execute('SELECT * FROM "{tn}" WHERE SourceTimestamp >= ? AND SourceTimestamp < ?'
```

Comment:
My understanding of the protocol is that returned messages are half bound so this code is updating "test servers" to reflect that.